src/controllers/auth.controller.js

```diff
@@ -0,0 +1,49 @@
+const userService = require("../services/user.database.service");
+const passwordService = require("../services/password.database.service");
+const tokenService = require("../services/token.database.service");
+
+const authLogin = async (req, res) => {
+  try {
+    if (req.body.email && req.body.password) {
+      //check the email do exist
+      const foundUser = await userService.findUserByEmail({
+        email: req.body.email,
+      });
+      if (!foundUser) {
+        res
+          .status(401)
+          .json({ status: false, message: "Email Does Not Exist" });
+      } else {
+        //check the password
+        const checkPassword = await passwordService.comparePassword(
```

Comment:
variable naming is wrong. since this is a boolean it should start with something liks `is` or `has`
eg: `isValid`